src/planner/builder/expr.go

```diff
@@ -388,30 +388,30 @@ func getTbsInExpr(expr sqlparser.Expr) []string {
 //  select b from (select a+1 as tmp,b from t1)t where tmp > 1;
 // If want to replace `tmp>1` with the fields in subquery.
 // The `colMap` is built from `a+1 as tmp` and `b`.
-// The `info` is built from `tmp>1`.
+// The `root` is built from `tmp>1`.
 // We need find and replace the cols from colMap.
 // Finally, `tmp > 1` will be overwritten as `a+1 > 1`.
-func replaceCol(info exprInfo, colMap map[string]selectTuple) (exprInfo, error) {
+func replaceCol(root exprInfo, colMap map[string]selectTuple) (exprInfo, error) {
 	var tables []string
 	var columns []*sqlparser.ColName
-	for _, col := range info.cols {
-		field, err := getMatchedField(col.Name.String(), colMap)
+	for _, old := range root.cols {
```

Comment:
`old` here is a column name, I think we add a comment "old is refer to a col name here".
I like name `from` more to keep consistent with `func ReplaceExpr(root, from, to Expr) Expr {...}`.